src/CovidSim.cpp

```diff
@@ -71,9 +71,10 @@ param P;
 person* Hosts;
 household* Households;
 popvar State, StateT[MAX_NUM_THREADS];
-cell* Cells; // Cells[i] is the i'th cell
-cell ** CellLookup; // CellLookup[i] is a pointer to the i'th populated cell
-microcell* Mcells, ** McellLookup;
+std::vector<cell> Cells(P.NC); // Cells.at(i) is the i'th cell
+std::vector<cell*> CellLookup(P.NCP); // CellLookup[i] is a pointer to the i'th populated cell
+std::vector<microcell> Mcells(P.NMC);
+std::vector<microcell*> McellLookup(P.NMCP);
```

Comment:
You are trying to initialise these at program start up time when `P.NC` is `0` - so this will fail - I think need to have something like:

```c++
std::vector<cell> Cells;

// Whereever the calloc was
Cells.resize(P.NC);
```

etc.